prometheus/alerts/missing_all_data.yaml

```diff
@@ -0,0 +1,25 @@
+groups:
+- name: slo-missing-all-data
+  rules:
+
+  # This is a generic alert which fires when there are no SLO data present
+  # and thanos-rule is having trouble evaluating rules.
```

Comment:
just for the record - it would not be "thanos". But for a simplicity I prefer to keep it here as it is.